.githooks/pre-push

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env sh
+
```

Comment:
It doesn't make a difference in this script but I always try to include `set -euo pipefail` in case the script starts to grow. This also requires writing `bash` instead of `sh` above.